cmake/BLTOptions.cmake

```diff
@@ -58,6 +58,8 @@ option(ENABLE_FORTRAN      "Enables Fortran compiler support" OFF)
 option(ENABLE_SHARED_LIBS  "Enables shared libraries." OFF)
 
 option(ENABLE_MPI          "Enables MPI support" OFF)
+option(ENABLE_FIND_MPI     "Enables CMake's Find MPI support (Turn off when compiling with the mpi wrapper directly)" ON)
```

Comment:
This should be an advanced option since most users will not have to care about this.

Reply:
Side note: We should think about how we name sub-options for how we use enabled packages, since we are starting to get a lot of options.